src/containers/blocks.jsx

```diff
@@ -656,18 +657,6 @@ Blocks.defaultOptions = {
         length: 2,
         colour: '#ddd'
     },
-    colours: {
-        workspace: '#F9F9F9',
-        flyout: '#F9F9F9',
-        toolbox: '#FFFFFF',
-        toolboxSelected: '#E9EEF2',
-        scrollbar: '#CECDCE',
-        scrollbarHover: '#CECDCE',
-        insertionMarker: '#000000',
-        insertionMarkerOpacity: 0.2,
-        fieldShadow: 'rgba(255, 255, 255, 0.3)',
-        dragShadowOpacity: 0.6
-    },
```

Comment:
Does removing this mean there should be a `.isRequired` on `Blocks.propTypes.options.colours`?

Reply:
Good point... Technically if it is left off the default `scratch-blocks` colors are used, which is nice. But make-toolbox-xml now expects certain colors to be provided. I've gone a different direction that I think accomplishes the same goal. I've passed the theme name to `<Blocks />`, including a default if it is not provided. This should ensure all of the colors the editor uses will be available to this component. See the latest commit.